bambam.py

```diff
@@ -171,8 +171,8 @@ def input(self, events, quit_pos):
             elif event.type == KEYDOWN or event.type == pygame.JOYBUTTONDOWN:
                 # check for words like quit
                 if event.type == KEYDOWN:
-                    if self.is_latin(event.key):
```

Comment:
Since `self.is_latin` is now unused, please remove it.

Reply:
done